io-engine-tests/Cargo.toml

```diff
@@ -0,0 +1,91 @@
+[package]
+name = "io-engine-tests"
+description = "I/O test support library"
+version = "1.0.0"
+edition = "2018"
+authors = [
```

Comment:
nit: it's no longer recommended to add authors to crates, and it might become deprecated and removed in the future.

Reply:
I'll remove it.